extensions/wikia/Search/classes/QueryService/Select/AbstractSelect.php

```diff
@@ -3,159 +3,182 @@
  * Class definition for Wikia\Search\QueryService\Select\AbstractSelect
  */
 namespace Wikia\Search\QueryService\Select;
+
 use Wikia\Search\QueryService\DependencyContainer, Wikia\Search\Config, \Solarium_Client, Wikia\Search\ResultSet, Wikia\Search\Utilities, \Solarium_Query_Select, \Solarium_Result_Select;
+
 /**
  * Abstract class responsible for controlling the flow of logic of a search select query.
  * The workflow for a search includes preparation of the query, sending of the query, and preparation of results.
+ *
  * @author relwell
  * @package Search
  * @subpackage QueryService
  */
-abstract class AbstractSelect
-{
+abstract class AbstractSelect {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "AbstractSelect" has 24 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)